js/admin/src/addProfileConfigurePane.js

```diff
@@ -5,7 +5,7 @@ import ProfileConfigurePane from 'flagrow/masquerade/panes/ProfileConfigurePane'
 
 export default function () {
     // create the route
-    app.routes['flagrow-masquerade-configure-profile'] = {path: '/flagrow/masquerade/configure', component: ProfileConfigurePane.component()};
+    app.routes['flagrow-masquerade-configure-profile'] = {path: '/masquerade', component: ProfileConfigurePane.component()};
```

Comment:
all our extensions have prefixed routes in the admin. I think we should keep `flagrow/`. However we can indeed drop the configure part given there is no other page, not even planned I guess